README.md

```diff
@@ -46,32 +38,26 @@ Add your own profile to nextflow.config file including all inputs
 nextflow run EBI-Metagenomics/motus_pipeline \
 -profile <choose profile> \
 --mode <single/paired> \
---readsdir <path to folder with fastq files> \
---sample_name <fastq filename>
+--single_end  / --paired_end_forward --paired_end_reverse <path with fastq file/s>\
+--sample_name <accession/name>
```

Comment:
Do we want to keep the sample_name?

Reply:
We are using it to build filenames. I think it makes sense to keep to simplify things.

Reply:
:+1: :1st_place_medal: